wiley/manifest.json

```diff
@@ -10,49 +10,94 @@
+      "aocs.onlinelibrary.wiley.com",
+      "aornjournal.onlinelibrary.wiley.com",
+      "aps.onlinelibrary.wiley.com",
+      "asbmr.onlinelibrary.wiley.com",
+      "ascpt.onlinelibrary.wiley.com",
+      "asistdl.onlinelibrary.wiley.com",
+      "aslopubs.onlinelibrary.wiley.com",
+      "aspenjournals.onlinelibrary.wiley.com",
+      "associationofanaesthetists-publications.onlinelibrary.wiley.com",
+      "awwa.onlinelibrary.wiley.com",
+      "bera-journals.onlinelibrary.wiley.com",
+      "besjournals.onlinelibrary.wiley.com",
+      "beva.onlinelibrary.wiley.com",
+      "bjssjournals.onlinelibrary.wiley.com",
+      "bjui-journals.onlinelibrary.wiley.com",
+      "bpspsychub.onlinelibrary.wiley.com",
+      "bpspubs.onlinelibrary.wiley.com",
+      "bsapubs.onlinelibrary.wiley.com",
+      "bsppjournals.onlinelibrary.wiley.com",
+      "ceramics.onlinelibrary.wiley.com",
+      "chemistry-europe.onlinelibrary.wiley.com",
+      "conbio.onlinelibrary.wiley.com",
+      "currentprotocols.onlinelibrary.wiley.com",
+      "dom-pubs.onlinelibrary.wiley.com",
+      "ebs.onlinelibrary.wiley.com",
+      "efsa.onlinelibrary.wiley.com",
+      "ejpr.onlinelibrary.wiley.com",
+      "esajournals.onlinelibrary.wiley.com",
+      "esj-journals.onlinelibrary.wiley.com",
+      "faseb.onlinelibrary.wiley.com",
+      "febs.onlinelibrary.wiley.com",
+      "headachejournal.onlinelibrary.wiley.com",
+      "iaap-journals.onlinelibrary.wiley.com",
+      "ietresearch.onlinelibrary.wiley.com",
+      "ifst.onlinelibrary.wiley.com",
+      "ila.onlinelibrary.wiley.com",
+      "iubmb.onlinelibrary.wiley.com",
+      "jlb.onlinelibrary.wiley.com",
+      "londmathsoc.onlinelibrary.wiley.com",
+      "mnrasl.oxfordjournals.org",
+      "movementdisorders.onlinelibrary.wiley.com",
+      "nasenjournals.onlinelibrary.wiley.com",
+      "ngwa.onlinelibrary.wiley.com",
+      "nph.onlinelibrary.wiley.com",
+      "nyaspubs.onlinelibrary.wiley.com",
+      "obgyn.onlinelibrary.wiley.com",
+      "ondmathsoc.onlinelibrary.wiley.com",
+      "onlinelibrary.wiley.com",
+      "physoc.onlinelibrary.wiley.com",
+      "rai.onlinelibrary.wiley.com",
+      "rgs-ibg.onlinelibrary.wiley.com",
+      "rmets.onlinelibrary.wiley.com",
+      "rsaiconnect.onlinelibrary.wiley.com",
+      "rss.onlinelibrary.wiley.com",
+      "sajournals.onlinelibrary.wiley.com",
+      "setac.onlinelibrary.wiley.com",
+      "sfamjournals.onlinelibrary.wiley.com",
+      "sigmapubs.onlinelibrary.wiley.com",
+      "sp.onlinelibrary.wiley.com",
+      "spssi.onlinelibrary.wiley.com",
+      "srcd.onlinelibrary.wiley.com",
+      "stemcellsjournals.onlinelibrary.wiley.com",
+      "sttipubs.onlinelibrary.wiley.com",
+      "theoncologist.onlinelibrary.wiley.com",
+      "wchh.onlinelibrary.wiley.com",
+      "wildlife.onlinelibrary.wiley.com",
+      "www.birpublications.org",
+      "www.embopress.org",
+      "www.onlinelibrary.wiley.com",
+      "www.tandfonline.com",
```

Comment:
tandfonline.com is not a wiley domain so this should probably be removed before the pull request is merged.

Reply:
Sorry, I will remove.  One of the journals in the PKB was transferred to Taylor and Francis in 2018 so the link from Wiley's site redirects there.  Didn't notice the change.  I will update the pull request to remove that line and 3 other transferred domains again, wiley link redirected to these sites.  Tried to be too clever